.github/workflows/main.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           path: |
             .cache
             public
-          key: ${{ runner.os }}-build-v1-${{ hashFiles('**/yarn.lock') }}
+          key: ${{ runner.os }}-build-v3-${{ hashFiles('**/yarn.lock') }}
```

Comment:
why this?

Reply:
As I said, I had some cache issue during github build, mainly related with graphql schema changes. Changing the cache key, the problem disappeared.